src/olympia/devhub/tests/test_views.py

```diff
@@ -665,7 +666,7 @@ def test_unlisted_addons_dashboard(self):
         self.add_log()
         res = self.client.get(reverse('devhub.addons'))
         doc = pq(res.content)
-        assert len(doc('.recent-activity li.item')) == 1
+        assert len(doc('.recent-activity li.item')) == 2
```

Comment:
As with the rest: the `ActivityLog` was previously tied to the current user, which was `None` in those tests, and so we didn't record one. In this test (and the others I've had to change) the task user is present, either explicitly or through a fixture, and suddenly we started recording the `ActivityLog` because I've tied it to the task user.